src/server_manager/messages/master_messages.json

```diff
@@ -501,6 +501,22 @@
     "message": "Click 'Create'.",
     "description": "This string appears within the server setup view as an item of a list that provides instructions to configure a firewall in Google Cloud Platform. 'Create' should be translated and included untranslated in parentheses."
   },
+  "hats_data_limits_title": {
```

Comment:
I have some suggestions on the wording here that I sent over the email thread with UX.

Reply:
Done.